security-analysis/security-analysis-api/src/main/java/com/powsybl/security/execution/SecurityAnalysisExecutionBuilder.java

```diff
@@ -44,7 +44,7 @@ public SecurityAnalysisExecutionBuilder(Supplier<ExternalSecurityAnalysisConfig>
                                             String providerName,
                                             SecurityAnalysisInputBuildStrategy inputBuildStrategy) {
         this.externalConfig = Objects.requireNonNull(externalConfig);
-        this.providerName = Objects.requireNonNull(providerName);
```

Comment:
And then I needed to change that to avoid crashing here, and to make the constructor consistent with its javadoc (which says that null is allowed)